_posts/2017-11-23-dnf-performance-boost.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "DNF performance boost"
+date:   2017-11-23 14:00:00 +0100
+author: Jaroslav Mracek <jmracek@redhat.com>
+visible: 1
+categories: dnf
+---
+Since the last blog there were several releases that enhance functionality and and mostly
+performance especially visible with multiple package arguments. Just try "dnf --assumeno reinstall

```

Comment:
asking users to do wrong commands doesn't seem to be right choice. People should use `dnf reinstall '*'`..